### PR TITLE
Sign in with ethereum

### PR DESCRIPTION
**Pull request by julien51**
# Description

Blog post to introdcuce Unlock's SIWE

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

## Release Note Draft Snippet

We now support SIWE!

### CODE REVIEW

**Comment by julien51** [comment]
sign-in-ethereum
0 0 0 0_3002_blog_sign-in-with-ethereum

**Comment by searchableguy** [comment]
This is super cool 😎

**Comment by desertrat10** [comment]
Looks great. Second paragraph, first sentence-- spell out "2" (anything under 10 should be spelled out-- all AP Style like) and the same sentence "as an objects" Object needs to be singular in that context.

File: unlock-protocol-com/blog/sign-in-with-ethereum.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+subTitle: The easiest way to let your users connect to your applications!
+authorName: Julien Genestoux
+publishDate: Feb 4 2022
+description: Sign-In with Ethereum is a new form of authentication that enables users to control their digital identity through their crypto-wallets.
+image: /images/blog/sign-in-ethereum/sign-in-ethereum.gif
+---
+
+We believe **everyone's online identity should not be tied to a platform**. Users must be able to authenticate using an identity that they fully own and control.
+
+There are 2 ways to think about physical wallets: as an objects that stores bills and coins but also as an object where identity cards, driving licenses and membership cards live. In the web3 space, we focus too much on the wallet as _cash holder_ and not enough on the wallet as _identity holder_, even though, in pratice, the private keys that power them are what enables the cash transfers through transaction signing.
```
Comment (julien51) [inline unlock-protocol-com/blog/sign-in-with-ethereum.md:12]:
```suggestion
There are two ways to think about physical wallets: as an objects that stores bills and coins but also as an object where identity cards, driving licenses and membership cards live. In the web3 space, we focus too much on the wallet as _cash holder_ and not enough on the wallet as _identity holder_, even though, in pratice, the private keys that power them are what enables the cash transfers through transaction signing.
```

File: unlock-protocol-com/blog/sign-in-with-ethereum.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+subTitle: The easiest way to let your users connect to your applications!
+authorName: Julien Genestoux
+publishDate: Feb 4 2022
+description: Sign-In with Ethereum is a new form of authentication that enables users to control their digital identity through their crypto-wallets.
+image: /images/blog/sign-in-ethereum/sign-in-ethereum.gif
+---
+
+We believe **everyone's online identity should not be tied to a platform**. Users must be able to authenticate using an identity that they fully own and control.
+
+There are two ways to think about physical wallets: as an objects that stores bills and coins but also as an object where identity cards, driving licenses and membership cards live. In the web3 space, we focus too much on the wallet as _cash holder_ and not enough on the wallet as _identity holder_, even though, in pratice, the private keys that power them are what enables the cash transfers through transaction signing.
```
Comment (julien51) [inline unlock-protocol-com/blog/sign-in-with-ethereum.md:12]:
```suggestion
There are two ways to think about physical wallets: as an object that stores bills and coins but also as an object where identity cards, driving licenses and membership cards live. In the web3 space, we focus too much on the wallet as _cash holder_ and not enough on the wallet as _identity holder_, even though, in pratice, the private keys that power them are what enables the cash transfers through transaction signing.
```

**Pull request merged**